README.md

```diff
@@ -7,7 +7,7 @@ ably-python
 
 ## Overview
 
-This is a Python client library for Ably. The library currently targets the [Ably 1.1 client library specification](https://ably.com/docs/client-lib-development-guide/features).
+This is a Python client library for Ably. The library currently targets the [Ably 2.0 client library specification](https://ably.com/docs/client-lib-development-guide/features).
```

Comment:
Probably worth updating this to avoid the redirect now.

```suggestion
This is a Python client library for Ably. The library currently targets the [Ably 2.0 client library specification](https://sdk.ably.com/builds/ably/specification/main/features/).
```